EditorFeatures tests are getting downgraded versions of Microsoft.CodeAnalysis.Test.Resources.Proprietary

Editor Features tests are getting downgraded versions of test package:
```
Restoring packages for D:\roslyn\src\EditorFeatures\CSharpTest2\project.json...
WARNING: Detected package downgrade: Microsoft.CodeAnalysis.Test.Resources.Proprietary from 1.2.0-beta-20151016-11 to 1.1.0-beta1-20150824-02
 CSharpEditorServicesTest2 (≥ 1.0.0) -> D:\roslyn\src\Test\Utilities\Desktop\TestUtilities.Desktop.csproj  -> Microsoft.CodeAnalysis.Test.Resources
.Proprietary (≥ 1.2.0-beta-20151016-11)
 CSharpEditorServicesTest2 (≥ 1.0.0) -> Microsoft.CodeAnalysis.Test.Resources.Proprietary (≥ 1.1.0-beta1-20150824-02)
WARNING: Detected package downgrade: Microsoft.CodeAnalysis.Test.Resources.Proprietary from 1.2.0-beta-20151016-11 to 1.1.0-beta1-20150824-02
 CSharpEditorServicesTest2 (≥ 1.0.0) -> D:\roslyn\src\Test\Utilities\Portable.FX45\TestUtilities.FX45.csproj  -> Microsoft.CodeAnalysis.Test.Resour
ces.Proprietary (≥ 1.2.0-beta-20151016-11)
 CSharpEditorServicesTest2 (≥ 1.0.0) -> Microsoft.CodeAnalysis.Test.Resources.Proprietary (≥ 1.1.0-beta1-20150824-02)
```

No idea what affect this has, but it's dirtying the logs and making builds harder to diagnose.

Related to this, looks like it's also causing this conflict when building:

```
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versio
ns of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to
detailed. [D:\roslyn\src\EditorFeatures\CSharpTest2\CSharpEditorServicesTest2.csproj]
```

--- a/src/EditorFeatures/CSharpTest2/project.json
+++ b/src/EditorFeatures/CSharpTest2/project.json
@@ -1,6 +1,5 @@
 ﻿{
   "dependencies": {
-    "Microsoft.CodeAnalysis.Test.Resources.Proprietary": "1.1.0-beta1-20150824-02",
     "Moq": "4.2.1402.2112",
     "System.Diagnostics.Process": "4.0.0-beta-23123",
     "xunit": "2.1.0",

